src/backend/base/langflow/components/embeddings/AzureOpenAIEmbeddings.py

```diff
@@ -77,7 +77,7 @@ def build_embeddings(self) -> Embeddings:
                 dimensions=self.dimensions or None,
             )
         except Exception as e:
-            msg = f"Could not connect to AzureOpenAIEmbeddings API: {str(e)}"
+            msg = f"Could not connect to AzureOpenAIEmbeddings API: {e}"
```

Comment:
The error could probably be removed from the message as it is propagated in the traceback ?

Reply:
In this case it is best not to change because this error is shown on the UI.